### PR TITLE
Add malaria access level for madagascar.

### PR DESCRIPTION
**Pull request by jonathansberry**
Add malaria access level to the local dev db.  This is done by adding three seperate levels: 

cd -- malaria
     \ 
        other

This is what I'm proposing for now - we will track to see what their access needs are moving forwards.

### CODE REVIEW

File: local_db.py [hunk 1]
```diff
@@ -326,8 +322,67 @@
...
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
+            ['madagascar', 'madagascar', 'madagascar'],
+            ['download', 'admin', 'cd'],
+            data={'name': {'value': 'Central Administrator'}},
+            state='new'
+        ), User(
+            'malaria',
+            'malaria@madagascartest.org.uk',
+            ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
```
Comment (tomeksabala) [inline local_db.py:376]: Password hashes in a public repo? Doesn't look like a good idea.

Comment (jonathansberry) [reply local_db.py:376]: This is just for the dev db.  It's fine - the password is just "password" and it doesn't give you access to any live data or information.

Comment (tomeksabala) [reply local_db.py:376]: My bad. I've only looked at file name `local_db.py` and thought it the local dynamodb setup. The domain `madagascartest.org.uk` should also be a good hint for me. Thanks!

**Review by tomeksabala — APPROVED** [review]

File: local_db.py (hunk 1 above)
Comment (jonathansberry) [reply local_db.py:376]: To be clear- We currently create a "fake" dev db locally for auth (and hermes actually) that allows us to test out and play with the auth functionality without ever infringing on live data - this script local_db.py sets that db up and creates a number of useful accounts roles that all have the same password: "password".

**Pull request merged**